recipe-server/normandy/recipes/api/v2/serializers.py

```diff
@@ -18,6 +18,10 @@
 
 class ActionSerializer(serializers.ModelSerializer):
     arguments_schema = serializers.JSONField()
+    # NOTE(peterbe): We *could* make the serializer NOT include
+    # this field if the action has no 'implementation'.
+    # Arguably it's nice when APIs are consistent with the keys.
+    # If the action does not have an implementation, this value will be None.
```

Comment:
I played for a bit with the idea of NOT including this field if the Action doesn't have an implementation. Just got deeper in the rabbit hole with little success. Then, upon reflection I feel more strongly about it always being included and left as `None` if there is no implementation. 
Worth considering deleting this comment.